### ISSUE TITLE
Different learning rates for encoder/decoder

### ISSUE THREAD

**Issue by JohnGiorgi** [JohnGiorgi/t2t#4]
Because our encoder is pre-trained, we select a much lower learning rate than usual (e.g. `2e-5`) a high weight decay (e.g. `0.1`) and train only for a few epochs.

Because all other layers are trained from scratch, they will likely benefit from a higher learning weights and smaller weight decay. 

In general, we should set the optimizer up to use different learning rate / weight decay's for the pre-trained weights and the weights trained from scratch.

**Comment by JohnGiorgi** [comment]
This no longer makes sense as we dropped the decoder.
